cmd/terramate/cli/cli.go

```diff
@@ -958,7 +954,7 @@ func (c *cli) filterStacksByWorkingDir(stacks []terramate.Entry) []terramate.Ent
 	return filtered
 }
 
-func newGit(basedir string, inheritEnv bool, checkrepo bool) (*git.Git, error) {
+func newGit(basedir string, checkrepo bool) (*git.Git, error) {
```

Comment:
Need to take a look on the linter cfg, it seems the default doesn't catch unused params like this on in this function :thinking: